tests/supervisor_purge_free_sims_test.py

```diff
@@ -67,16 +67,14 @@ def _status_eq(next_req, status):
     next_req_premium = _run_sim(fc.sr_sim_data())
     fc.sr_email_login(user_free)
     next_req_free = _run_sim(fc.sr_sim_data())
-    fc.sr_get_json(
-        "adjustTime",
-        params=PKDict(days=_PURGE_FREE_AFTER_DAYS + 1),
-    )
+    _adjust_time(fc)
     time.sleep(_CACHE_AND_SIM_PURGE_PERIOD + 1)
     _status_eq(next_req_free, "job_run_purged")
     _check_run_dir(should_exist=0)
     fc.sr_email_login(user_premium)
     _status_eq(next_req_premium, "completed")
     _check_run_dir(should_exist=7)
+    _adjust_time(fc, forward=False)
```

Comment:
Yes, that's right. I commented to that effect but it's not in the commit message. Is it OK to make a "meaningless" commit and include the reasoning in that message?

Reply:
I see https://github.com/radiasoft/sirepo/pull/4846#discussion_r997667401 now. No need for a commit message. It's part of the logic now. I would change "forward" to "restore" and invert the logic, since that's what's happening. I didn't understand why it had to be moved back.